examples/associationsetup/ac-client/main.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019-2020 go-pfcp authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+package main
```

Comment:
You need one more empty line so that the Copyright notice is not interpreted as GoDoc(same for other files).

Also, could you add brief description of this command like [this](https://github.com/wmnsk/go-pfcp/blob/master/examples/heartbeat/hb-client/main.go#L5-L8)?